src/screens/Homepage.tsx

```diff
@@ -1,12 +1,102 @@
 import React from 'react'
-import { Box } from '@chakra-ui/react';
+import { Box, Badge, Text, Center, Image, Flex, Button, Link } from '@chakra-ui/react';
+import Slider from "../components/Slider";
+import { FiTruck } from 'react-icons/fi'
+import { BiMessageDetail } from 'react-icons/bi'
+import { RiSecurePaymentLine } from 'react-icons/ri'
+import Card from '../components/Card'
+
+
+const pargraph = 'Contrary popular belief Lorem Ipsu not simply random tex';
+
 
 
 const Homepage = () => {
+    const dataList = [
+        {
+            id: "1",
+            product: "Product 1",
+            summary: "This is a summary, can be any length",
+            longLine: "Very short, can be any description"
+        },
+        {
+            id: "2",
+            product: "Product Two",
+            summary:
+                "Another summary, make sure that this is very responsivesfafsdfsdfsdfsdfsfsfsdf",
+            longLine: "Billy Bob Bob Bob Bob likes Markiplier gameplay videos"
+        },
+        {
+            id: "3",
+            product: "Long Product",
+            summary: "Finalize them summary, hurry, we are close to deadline",
+            longLine: "Wow, this is very descriptive! I wonder how long it is"
+        }
+    ];
     return (
-        <Box>
-            Homepage
-        </Box>
+        <>
+
+            <Box p={'4'} bg={'#F5F5F5'}>
+                <Center >
+                    <Flex alignItems={'center'}>
+                        <Badge mx={'10px'} bg={'blue.600'} color={'#FFF'}>OFFER</Badge>
+                        <Text><b>Need It Today?</b> Free In Store & Pickup / <b>$9.99</b> Same Day Delivery</Text>
+                    </Flex>
+                </Center>
+            </Box>
```

Comment:
Simple, always when you write React components think about it as COMPONENTS, every tiny jsx or html can be a component. okay when we can create a component and when we can not. 

Basically we always can, as long as this component can be a piece or a unit. for example here. do you think having it like this or extracting this into a new component that can be name as Banner for example? 

```suggestion
            <Banner />
```

and then you can have the banner component like this 

```tsx
const Banner = () => {
return ( 
  <Box p={'4'} bg={'#F5F5F5'}>
                <Center >
                    <Flex alignItems={'center'}>
                        <Badge mx={'10px'} bg={'blue.600'} color={'#FFF'}>OFFER</Badge>
                        <Text><b>Need It Today?</b> Free In Store & Pickup / <b>$9.99</b> Same Day Delivery</Text>
                    </Flex>
                </Center>
     </Box>
  )
}
```

then we can make this component a re-usable one also. some dynamic data to be passed. like the text for example. but for now we can just move it out

Reply:
I thought when less component we use it's better.
GONNA UPDATE IT !!